### DISCUSSION TITLE
JWKS not been cached since 4.8.0

### DISCUSSION THREAD

**Discussion by tbashour** [micronaut-projects/micronaut-security#1781] [Q&A]
Hi,

In micronaut-security the caching of JWKs was implemented in the case JwksSignature. But since the class is deprecated since 4.8.0, the caching will not work, right? am I missing something?

**Comment by sdelamo** [comment]
it has been restored via https://github.com/micronaut-projects/micronaut-security/pull/1824 It will be part of Micronaut Seuciry 4.11.0
